### PR TITLE
Fix bug on login page when password field is empty

### LINKED ISSUE #1280: Login button is enabled even if password field is empty.
**Actual Behaviour**
example

**Expected Behaviour**
example1

**Steps to reproduce it**
On the login page enter any email address while keeping the password field empty.

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by PrP-11**
Fixes #1280 

Changes: Fixed bug on login page. Earlier login button is enable even if password field is empty. Now we need to enter both email and password to enable it.

Demo Link: https://pr-1281-fossasia-susi-web-chat.surge.sh

Screenshots for the change: 

Earlier
example

Now
example1

### CODE REVIEW

File: src/components/Auth/Login/Login.react.js [hunk 1]
```diff
@@ -207,7 +207,9 @@ class Login extends Component {
 
 	    if (!state.emailError && !state.passwordError && !state.serverFieldError)
 	    {
-	    	state.validForm = true;
+		if(this.state.password !== '' && this.state.email !== ''){
```
Comment (DNS-404) [inline src/components/Auth/Login/Login.react.js]: How about you add this condition to the above condition?
Like this - 
```
if (!state.emailError && !state.passwordError && !state.serverFieldError && this.state.password !== '' && this.state.email !== '')
```

**Comment by PrP-11** [comment]
@Akshat-Jain Its already a single commit.

File: src/components/Auth/Login/Login.react.js (hunk 1 above)
Comment (PrP-11) [reply src/components/Auth/Login/Login.react.js]: Done @DNS-404

**Comment by Akshat-Jain** [comment]
@PrP-11 There are 2 commits. Please squash them. Thanks :-)
2commits

**Pull request merged**